work/v1/types.go

```diff
@@ -79,9 +79,55 @@ type ManifestConfigOption struct {
 	ResourceIdentifier ResourceIdentifier `json:"resourceIdentifier"`
 
 	// FeedbackRules defines what resource status field should be returned.
+	// +optional
+	FeedbackRules []FeedbackRule `json:"feedbackRules,omitempty"`
+
+	// UpdateStrategy defines the strategy to update this manifest
+	// optional
```

Comment:
Please clarify if there is a difference between missing and empty (there shouldn't be a difference without a really good reason).

Reply:
coment added